lib/designateDevice.js

```diff
@@ -10,11 +10,11 @@ function handleDesignateDevice(request, response) {
     const alertApiKey = request.header('X-API-KEY')
 
     helpers.log(`************* Verification Code: ${verificationCode} Alert API Key: ${alertApiKey} Responder Push ID: ${responderPushId}`)
-    response.status(200).json(JSON.stringify('OK'))
```

Comment:
This doesn't actually need the `JSON.stringify(...)` to work with the Alert App. So I took it out to simplify things.

Reply:
agreed. `response.json()` already stringifies the argument, so we shouldn't need to stringify it again.

Reply:
OK, I think I've removed all the excess `stringify` calls now.